browser-test/src/applicant/questions/name.test.ts

```diff
@@ -217,6 +217,42 @@ test.describe('name applicant flow', () => {
     })
   })
 
+  test.describe('name question with name suffix flag enabled', () => {
+    const programName = 'Test program for name suffix'
+
+    test.beforeEach(async ({page, adminQuestions, adminPrograms}) => {
+      await setUpSingleRequiredQuestion(
+        programName,
+        page,
+        adminQuestions,
+        adminPrograms,
+      )
+      await enableFeatureFlag(page, 'name_suffix_dropdown_enabled')
+    })
+
+    test('name questions with suffix field being available to use', async ({
+      page,
+      applicantQuestions,
+    }) => {
+      await applicantQuestions.applyProgram(programName)
+
+      await test.step('opens name question during application process', async () => {
+        await validateScreenshot(
+          page.locator('.cf-question-name'),
+          'name-question-with-suffix-field',
+        )
```

Comment:
```suggestion
        await expect(page.getByLabel('Suffix')).toBeVisible()
        await expect(page.getByLabel('Suffix')).toHaveValue('')
        
        // TODO: I don't recall if you can chain assertions, but if you can I'd do this
        // await expect(page.getByLabel('Suffix')).toBeVisible().toHaveValue('')
```

(recommended) Screenshots are a lot heavier process than asserting other things. Unless you really need screenshots here it's be better using other [locator assertions](https://playwright.dev/docs/api/class-locatorassertions). 

The above suggestion also looks up the control by it's accessible name which is what playwright recommends as that is the same way screen readers would find it.